### DISCUSSION TITLE
Division Operator issues in MoveHub Class

### DISCUSSION THREAD

**Discussion by alpha71** [pybricks/support#460] [General]
Hello guys,
i found that in the MoveHub class cannot use the divider operator '/' just can use the intger operator '//'! the division give out a error as _TypeError: unsupported type for operator_ . This happens in both version of Pybricks: Stable and Beta! Can you please check it.
Thank you

**Comment by BertLindeman** [comment]
Hello Allessandro,

The movehub has very limited memory aboard.
So a number of features that are available on other hubs are not implemented on the movehub.

And floating point is one of those features. 
See [support issue 383](https://github.com/pybricks/support/issues/383)

Hope this helps,

Bert

**Comment by alpha71** [comment]
Hello Bert,
Thank you , in fact I guessed so! I just feel weird because the CityHub
seems has more features than MoveHub (even though it is much poor for I/O!).
However thank you for you quick feedback .
Bye
Alessandro
